example/lib/03.control/widgets/code_editor_appbar.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+
+import 'dropdown_selector.dart';
+
+class CodeEditorAppbar extends StatelessWidget implements PreferredSizeWidget {
+  final double height;
+  final List<String?> languages;
+  final List<String?> themes;
+  final String? title;
+  final Function(String?)? onLanguageChanged;
+  final Function(String?)? onThemeChanged;
+
+  final String selectedLanguage;
+  final String selectedTheme;
+  
+  final VoidCallback? onReset;
+
+  const CodeEditorAppbar({
+    super.key,
+    required this.height,
+    required this.languages,
+    required this.themes,
+    required this.selectedLanguage,
+    required this.selectedTheme,
+    this.title,
+    this.onLanguageChanged,
+    this.onThemeChanged,
+    this.onReset,
```

Comment:
https://medium.com/flutter-senior/sort-alphabetically-everything-you-can-1e9d7299c4ae

Check everything else to sort in the app.